x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule_details/index.tsx

```diff
@@ -129,7 +129,7 @@ const StepAboutRuleToggleDetailsComponent: React.FC<StepPanelProps> = ({
                 {(resizeRef) => (
                   <AboutContent ref={resizeRef}>
                     <VerticalOverflowContainer maxHeight={120}>
-                      <VerticalOverflowContent maxHeight={120} className="eui-yScrollWithShadows">
+                      <VerticalOverflowContent maxHeight={120} className="eui-yScroll">
```

Comment:
Nit: `VerticalOverflowContent` is a styled-component, and we could specify the `eui-yScroll` class name in it for reusability and remove it from the 3 places where this component is rendered.